libcrux-psq/Cargo.toml

```diff
@@ -10,18 +10,19 @@ readme.workspace = true
 description = "Libcrux Pre-Shared post-Quantum key establishement protocol"
 publish = false
 
-[lib]
-path = "src/psq.rs"
-
 [dependencies]
-libcrux-kem = { version = "0.0.2-alpha.1", path = "../libcrux-kem", features = ["pre-verification"] }
+libcrux-kem = { version = "0.0.2-alpha.1", path = "../libcrux-kem", features = [
+    "pre-verification",
+] }
 libcrux-hkdf = { version = "=0.0.2-alpha.1", path = "../libcrux-hkdf" }
 libcrux-hmac = { version = "=0.0.2-alpha.1", path = "../libcrux-hmac" }
 classic-mceliece-rust = { version = "2.0.0", features = [
     "mceliece460896f",
     "zeroize",
 ] }
 rand = { version = "0.8" }
+libcrux-ecdh = { version = "0.0.2-alpha.1", path = "../libcrux-ecdh" }
+libcrux = { version = "0.0.2-alpha.1", path = ".." }
```

Comment:
I guess we should expose everything through the `libcrux` crate eventually so that we only need this one dependency. (Not for now or this PR.)